app/controllers/build_commands_controller.rb

```diff
@@ -11,7 +11,11 @@ def show
   def update
     command = params[:command][:command]
     if command.blank?
-      @command.destroy
+      Command.transaction do
+        current_project.update_attributes(build_command: nil)
+        @command.projects.reload
+        @command.destroy!
```

Comment:
could remove the transaction and do a destroy here ... that would then fail ... but we don't care ... and the build command would still be removed